Create ocis 7.0.0 release notes (#81)
owncloud/docs-main: modules/ROOT/pages/ocis_release_notes.adoc

* The following services have been added:

** xref:next@ocis:ROOT:{s-path}/activitylog.adoc[activitylog] +
It is responsible for storing events (activities) per resource. +
Note that the implementation of the `activitylog` service is a breaking change. For details see xref:breaking-changes[Breaking Changes] section.

** xref:next@ocis:ROOT:{s-path}/auth-app.adoc[auth-app] +
It provides authentication for 3rd party apps.

** xref:next@ocis:ROOT:{s-path}/collaboration.adoc[collaboration] +
It connects with web office document servers such as Collabora, ONLYOFFICE or Microsoft using the WOPI protocol. Note that the `collaboration` service replaces the external https://github.com/cs3org/wopiserver[cs3org/wopi] server used before. If you have used document servers with the external wopiserver, we highly recommend using the collaboration service which is a breaking change. For details see xref:breaking-changes[Breaking Changes] section.

* Content Security Policy: +
When using web office document servers or web apps like provided in our docker compose deployment examples, a CSP has been implemented to secure the environment. For details on web office/apps see the next list item. 

> * Fully customisable deployment examples ready for productive use including web office, based on docker compose. See the

Reviewer: ```suggestion
* Fully customisable deployment examples ready for production use including web office, based on docker compose. See the
```

=== Migrations

See the xref:next@ocis:ROOT:migration/upgrading-ocis.adoc[Upgrading Infinite Scale] documentation for migration and upgrade steps.

=== Breaking Changes

Infinite Scale::
+
--
* The xref:next@ocis:ROOT:{s-path}/activitylog.adoc[activitylog] service has been added. +
To make it work, the Infinite Scale config must be updated. See the link in the xref:migrations[migrations] section above for details how to do so.
* The xref:next@ocis:ROOT:{s-path}/collaboration.adoc[collaboration] service has been added. +
If you have used web office document servers with the external https://github.com/cs3org/wopiserver[cs3org/wopi] server, you should switch to the collaboration service. See the link in the xref:migrations[migrations] section above for details how to do so.
--

Web UI::
+
--
* Deprecated config options have finally been removed, see https://github.com/owncloud/web/pull/11256[#11256]. +
Note that these config options were able to be set via the `WEB_UI_CONFIG_FILE` environment variable. Having one of the deprecated set, will not have any impact.

> * The `draw-io` app has been removed as default Web app. It's now part of the https://github.com/owncloud/web-extensions[web-extensions repository].
> * The `importer` app has been removed as default Web app. It's now part of the https://github.com/owncloud/web-extensions[web-extensions repository].

Reviewer: ```suggestion
* The `draw-io` app has been removed as a default Web app. It's now part of the https://github.com/owncloud/web-extensions[web-extensions repository].
* The `importer` app has been removed as a default Web app. It's now part of the https://github.com/owncloud/web-extensions[web-extensions repository].
```

I suppose that there are multiple apps in the "list of default web apps", and these 2 apps have been removed from the list. Is that what is meant?

Note that the implementation of the `activitylog` service is a breaking change. For details see xref:breaking-changes[Breaking Changes] section.

** xref:next@ocis:ROOT:{s-path}/auth-app.adoc[auth-app] +
It provides authentication for 3rd party apps.

** xref:next@ocis:ROOT:{s-path}/collaboration.adoc[collaboration] +
It connects with web office document servers such as Collabora, ONLYOFFICE or Microsoft using the WOPI protocol. Note that the `collaboration` service replaces the external https://github.com/cs3org/wopiserver[cs3org/wopi] server used before. If you have used document servers with the external wopiserver, we highly recommend using the collaboration service which is a breaking change. For details see xref:breaking-changes[Breaking Changes] section.

* Content Security Policy: +
When using external services like an IDP, web office document servers or web apps (like provided in our docker compose deployment examples), a Content Security Policy (CSP) has been implemented to secure the environment. For details on web office/apps see the next list item. For details on CSP see xref:breaking-changes[Breaking Changes] section.

> * Fully customisable deployment examples ready for production use including web office, based on docker compose. See the

Reviewer: ```suggestion
* Fully customisable deployment examples ready for production use including web office, based on docker compose. See
```

xref:next@ocis:ROOT:depl-examples/ubuntu-compose/ubuntu-compose-prod.adoc[Local Production Setup] and xref:next@ocis:ROOT:depl-examples/ubuntu-compose/ubuntu-compose-hetzner.adoc[Deployment on Hetzner]. +
Note that with these examples, you can easily enable additional web apps like `draw-io`, `external-sites`, `json-viewer`, `unzip`, `progressbars` and others. +
Though provided via the compose example setup, no support can be given for these web apps.

* Additonal external web apps can be configured _manually_ to the deployment examples.
** https://github.com/owncloud/web-app-dicom-viewer[Dicom Viewer] +
The Dicom Viewer Web Extension offers an easy way to preview medical images of DICOM standard (.dcm) such as Ultrasound, X-ray, CT scan, MRI, including all their corresponding meta data. The app offers image manipulation operations such as zoom in and out, rotation, flipping, colour inversion on the image preview.

** https://github.com/saw-jan/web-app-3dmodel-viewer[3D Model Viewer] +

> The 3D model viewer Web Extension is an extension for ownCloud Web which offers preview of .glb, .stl, .fbx, and .obj files. The app allows to zoom and rotate the model, to navigate between model files, plus it supports full screen view.

Reviewer: ```suggestion
The 3D model viewer Web Extension is an extension for ownCloud Web which offers preview of .glb, .stl, .fbx, and .obj files. The app allows zooming and rotating the model, navigating between model files, plus it supports full screen view.
```

** http://github.com/JankariTech/web-app-presentation-viewer[Presentation Viewer] +

> The Presentation Viewer Web Extension allows to render markdown slide presentations from .md files with reveal.js. It can be used both for preview and for sharing the slides using public links.

Reviewer: ```suggestion
The Presentation Viewer Web Extension allows rendering markdown slide presentations from .md files with reveal.js. It can be used both for preview and for sharing the slides using public links.
```

> 

Reviewer 2: maybe mention that even more web extensions can be found at https://github.com/owncloud/awesome-ocis

Author: You will find them in the above list item.

Reviewer 2: not all of them... for example Dominic's GPX viewer is not mentioned

Author: Correct, but the app is not integrated as the rest of them (as you did with yours)... At the moment it is available, I do not see any objections adding it.

=== Breaking Changes

Infinite Scale::
+
--
* The xref:next@ocis:ROOT:{s-path}/activitylog.adoc[activitylog] service has been added. +
To make it work, the Infinite Scale config must be updated. See the link in the xref:migrations[migrations] section above for details how to do so.

* The xref:next@ocis:ROOT:{s-path}/collaboration.adoc[collaboration] service has been added. +
The Infinite Scale config must be updated. If you have used web office document servers with the external https://github.com/cs3org/wopiserver[cs3org/wopi] server, you should switch to the collaboration service. See the link in the xref:migrations[migrations] section above for details how to do so.

* A Content Security Policy (CSP) has been implemented to secure the environment. Its configuration is described in the xref:next@ocis:ROOT:{s-path}/proxy.adoc[proxy] service. Without configuration, external services like an IDP, web office document servers or web apps will refuse to work for security reasons.
--

Web UI::
+
--
* Deprecated config options have finally been removed, see https://github.com/owncloud/web/pull/11256[#11256]. +

> Note that these config options were able to be set via the `WEB_UI_CONFIG_FILE` environment variable. Having one of the deprecated set, will not have any impact.

Reviewer: ```suggestion
Note that these config options were able to be set via the `WEB_UI_CONFIG_FILE` environment variable. Having one of the deprecated options set, will not have any impact.
```

* Infinite Scale: {ocis-releases-url}/v7.0.0[Changes in 7.0.0, window=_blank]
+
NOTE: All changes from rolling releases 6.x apply for this production release.

> * Web: {web-releases-url}/v11.0.0[Changelog for ownCloud Web 11.0.3, window=_blank]

Reviewer 3: ```suggestion
* Web: {web-releases-url}/v11.0.3[Changelog for ownCloud Web 11.0.3, window=_blank]
```
